lib/ansible/modules/windows/win_say.ps1

```diff
@@ -36,7 +36,7 @@ $result = @{
 $speed = 0
 $words = $null
 
-if ($speech_speed -ne $null) {
+if ($speech_speed) {
```

Comment:
I rewrote this part, now it defaults to 0 and the normal flow is more readable !
Thanks for reviewing !